axon-server/administration/reset-event-processor-token.md

```diff
@@ -0,0 +1,146 @@
+# How to reset the Token of an Event Processor
+
+It might be desirable to reset the position an event processor works on.
+This is done by resetting its token.
+
+We document the following ways to reset the token of an event processor:
+- Using Axon Framework
```

Comment:
Note the reset invocation from Axon Framework is already described [here](https://docs.axoniq.io/reference-guide/axon-framework/events/event-processors/streaming#triggering-a-reset). It's not wrong to rehash it here, but I would definitely reference the already existing section.